pkg/util/file/fileutil.go

```diff
@@ -55,3 +56,27 @@ func (w crlfWriter) Write(b []byte) (n int, err error) {
 		i = next + 1
 	}
 }
+
+// loadData reads the specified file and return it as a bytes array.
+func loadData(file string) ([]byte, error) {
+	if len(file) == 0 {
+		return []byte{}, nil
+	}
+
+	bytes, err := ioutil.ReadFile(file)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	return bytes, nil
+}
+
+// LoadCert reads the specified certificate file and returns it as a bytes array.
+func LoadCert(file string) ([]byte, error) {
```

Comment:
Do you plan to add any logic specific to loading cert and key files here, like structure validation or something similar? Otherwise I don't see the valid in this being specific, should just use a generic loader like `LoadData` or `LoadFileData`.